src/librustc_mir/interpret/operand.rs

```diff
@@ -353,6 +353,31 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         Ok(str)
     }
 
+    // Turn the wide MPlace into a OsString (must already be dereferenced!)
+    #[cfg(target_os = "windows")]
```

Comment:
While this is probably only useful on Windows, I don't think it should be cfg-gated. Having these functions be compiled unconditionally will make it easier for people building on other platfirms to notice if they end up breaking those functions through other changes.

Reply:
Also, a Linux host might totally run a Windows target. So this should definitely be built for all platforms.